src/common/constants-ln.ts

```diff
@@ -0,0 +1 @@
+constant.ts
```

Comment:
Why did you create this link file?

Reply:
resolve the path conflict in dashboard.ts,  this file is used both in reporting and data modeling.

Reply:
I just see `dashboard.ts` is referred to in the backend. What's the problem that you met with?

Reply:
import { REPORTING_VIEW_VERSION } from '../../common/constants-ln';

Reply:
I don't see the reason why we need another link file on the same path.

Maybe you can extract those constants from the dashboard and save them to the constant file.